llvm/lib/MC/EVMBinaryObjectWriter.cpp

```diff
@@ -128,6 +128,10 @@ uint64_t EVMBinaryObjectWriter::writeObject(MCAssembler &Asm,
     Json.attribute("size", Size);
     Json.attribute("visible", Visible);
 
+    if (StringRef(FuncName).startswith("_GLOBAL__sub_I_")) {
```

Comment:
What does this prefix mean? A comment would be helpful (but a helper function with a meaningful name might be even better).

Reply:
This prefix is used by LLVM for global init function. Will add some comment https://github.com/NilFoundation/zkllvm-circifier/blob/48fd30c40daef638c21db6f7dd33e4d3f97438c9/clang/lib/CodeGen/CGDeclCXX.cpp#L897